### PR TITLE
25707/claims status cypress

### PR DESCRIPTION
**Pull request by CBonade**
## Description

Test rewrite converting from Nightwatch to Cypress for the claims-status tests.  There's a fair bit of comments in the code regarding upload functionality that was not supported by Nightwatch as well as things that were commented out until the COVID-19 messaging goes away.  These have been left as-is, in the spirit of keeping focused on strictly a conversion effort at this time.  We have an outstanding backlogged task, to re-review these tests in a future initiative, to make improvements to them.

## Testing done
All specs have been tested in the Cypress runner in Chrome.  All have made it to the end of their specs without breaking.

## Screenshots
Normally I would include screenshots of the tests passing here, however, there are 15 specs here so I felt 15 screenshots would be a bit overkill.

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/claims-status/tests/e2e/00.claims-list-breadcrumb.cypress.spec.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      '.va-nav-breadcrumbs-list li:nth-of-type(2) a[aria-current="page"]',
+    ).should('exist');
+    cy.get(
+      '.va-nav-breadcrumbs-list li:nth-of-type(2) a[aria-current="page"]',
+    ).should('contain', 'Check your claims and appeals');
+    cy.get(
+      '.va-nav-breadcrumbs-list li:nth-of-type(2) a[aria-current="page"]',
+    ).should('have.css', 'pointer-events', 'none');
+
+    cy.viewport(375, 667);
```
Comment (ddzz) [inline src/applications/claims-status/tests/e2e/00.claims-list-breadcrumb.cypress.spec.js:35]: Is this a mobile viewport? We are trying to move away from using hardcoded viewports and using the most common viewports instead with `cy.viewportPreset()`. See https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/testing/end-to-end/cypress-best-practices-on-vsp.md#viewport-presets-cyviewportpresetpreset-orientation-options

File: src/applications/claims-status/tests/e2e/00.claims-list-breadcrumb.cypress.spec.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+const Timeouts = require('platform/testing/e2e/timeouts.js');
+
+import claimsList from './fixtures/mocks/claims-list.json';
+
+describe('Breadcrumb Test', () => {
+  it('Verifies breadcrumb functionality', () => {
+    cy.intercept('GET', '/v0/evss_claims_async', claimsList);
+    cy.visit('/track-claims');
+    cy.login();
```
Comment (ddzz) [inline src/applications/claims-status/tests/e2e/00.claims-list-breadcrumb.cypress.spec.js:9]: Not sure if it matters here, but `cy.login()` usually comes before `cy.visit()`.

**Review by Mottie — APPROVED** [review]
Thanks!

File: src/applications/claims-status/tests/e2e/00.claims-list-breadcrumb.cypress.spec.js (hunk 1 above)
Comment (CBonade) [reply src/applications/claims-status/tests/e2e/00.claims-list-breadcrumb.cypress.spec.js]: Didn't know about these, but what a great idea to keep things up to date!  Went ahead and converted this to top mobile viewport #1

File: src/applications/claims-status/tests/e2e/00.claims-list-breadcrumb.cypress.spec.js (hunk 2 above)
Comment (CBonade) [reply src/applications/claims-status/tests/e2e/00.claims-list-breadcrumb.cypress.spec.js:8]: Checked and it works the same both ways, but went ahead and moved it in each test to before as if that's a standard, I don't want to be the one that undoes our norm!  Moved it in all the tests and they all still pass as they did before.

**Comment by pjhill** [comment]
Resolves [25705](https://github.com/department-of-veterans-affairs/va.gov-team/issues/25705)

**Pull request merged**